electron/package.json

```diff
@@ -16,8 +15,6 @@
     "webpack-cli": "^3.1.2"
   },
   "dependencies": {
-    "@tensorflow/tfjs-core": "^0.8.0",
-    "electron": "^3.0.4",
-    "tone-piano": "0.0.12"
+    "@tensorflow/tfjs-core": "^1.3.1"
```

Comment:
This caused some breaking changes to the API, so I'm trying to investigate that while refactoring the whole code with more proper types.